src/main/java/codeu/controller/ServerStartupListener.java

```diff
@@ -44,6 +45,8 @@ public void contextInitialized(ServletContextEvent sce) {
 
       NemoBot nemoBot = new NemoBot("../../src/main/java/codeu/model/data/NemoBot.txt");
       botController.registerBot(nemoBot.getMentionKey(), nemoBot);
+      ConversationStatBot conversationStatBot = new ConversationStatBot();
+      botController.registerBot(conversationStatBot.getMentionKey(), conversationStatBot);
```

Comment:
Did we never get around to making `registerBot` take only the Bot instance, and call its `getMentionKey` method within? I can't remember how that discussion went.

Reply:
I can update this. Is this what similar to what you have in mind?

```
botController.registerBot(bot); // In ServerStartUpListener
...
// In BotController
registerBot(Bot bot) {
botMap.put(bot.getMentionKey(), bot);
...
}
```

Reply:
Yup, exactly!